src/lib/models/batch.js

```diff
@@ -4,33 +4,18 @@ const FinancialYear = require('./financial-year');
 const { assert } = require('@hapi/hoek');
 const { isArray } = require('lodash');
 
-const VALID_GUID = Joi.string().guid().required();
 const VALID_BATCH_TYPE = Joi.string().valid('annual', 'supplementary', 'two_part_tariff').required();
 const VALID_SEASON = Joi.string().valid('summer', 'winter', 'all year').required();
 const VALID_STATUS = Joi.string().valid('processing', 'complete', 'error').required();
 
-class Batch {
+const Model = require('./model');
+
+class Batch extends Model {
   constructor () {
```

Comment:
To follow the pattern of the other models which don't have a constructor, do we need to pass the ID to the superclass:
```
constructor(id) {
   super(id);
} 
```
or
```
constructor(...args) {
   super(...args);
} 
```